rosdep/base.yaml

```
@@ -691,6 +691,12 @@ couchdb:
   gentoo: [dev-db/couchdb]
   nixos: [couchdb3]
   ubuntu: [couchdb]
+cpp-httplib:
+  alpine:
+    '*': [cpp-httplib-dev]
```

`cpp-httplib-dev` seems not have dependency to `cpp-httplib`, so
```suggestion
    '*': [cpp-httplib, cpp-httplib-dev]
```